### PR TITLE
Update dataset names

### PR DESCRIPTION
**Pull request by wpreimes**
C3S SMOS and SMAP names updated

### CODE REVIEW

**Comment by wpreimes** [comment]
todo

-[] Confirm new dataset names are ok
-[] Merge https://github.com/awst-austria/qa4sm-testdata/pull/8
-[] Merge https://github.com/awst-austria/qa4sm-reader/pull/46
-[] Update https://github.com/wpreimes/qa4sm/blob/update-dataset-names/environment/qa4sm_env.yml#L240 and testdata commit
-[] Change folder names on qa4sms1

**Comment by wpreimes** [comment]
image
image

File: validator/fixtures/datasets.json [hunk 1]
```diff
@@ -38,10 +38,10 @@
         "model": "validator.dataset",
         "pk": 2,
         "fields": {
-            "short_name": "SMAP",
-            "pretty_name": "SMAP level 3",
+            "short_name": "SMAP_L3",
+            "pretty_name": "SMAP SPL3SMP",
             "help_text": "NASA SMAP level 3 data",
-            "storage_path": "testdata/input_data/SMAP",
+            "storage_path": "testdata/input_data/SMAP_3",
```
Comment (sheenaze) [inline validator/fixtures/datasets.json:44]: shouldn't it be SMAP_L3?

**Comment by wpreimes** [comment]
So this is now ready from my side and can be merged

**Pull request merged**